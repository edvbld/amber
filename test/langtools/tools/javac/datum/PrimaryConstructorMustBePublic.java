/*
 * @test /nodynamiccopyright/
 * @summary smoke negative test for record classes
 * @compile/fail/ref=PrimaryConstructorMustBePublic.out -XDrawDiagnostics PrimaryConstructorMustBePublic.java
 */

public record PrimaryConstructorMustBePublic(int i) {
    PrimaryConstructorMustBePublic {}
}
