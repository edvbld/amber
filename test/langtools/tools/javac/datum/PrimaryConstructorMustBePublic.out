PrimaryConstructorMustBePublic.java:8:5: compiler.err.method.must.be.public: <init>
1 error
